x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/components/edit_integration_flyout.tsx

```diff
@@ -128,6 +166,42 @@ export const EditIntegrationFlyout: React.FunctionComponent<{
         </EuiFlexGroup>
       </EuiFlyoutHeader>
       <EuiFlyoutBody>
+        <EuiFormRow
+          fullWidth
+          label={
+            <FormattedMessage
+              id="xpack.fleet.epm.editIntegrationFlyout.categoriesLabel"
+              defaultMessage="Integration Categories"
+            />
+          }
+          helpText={
+            <FormattedMessage
+              id="xpack.fleet.epm.editIntegrationFlyout.categoriesHelpText"
+              defaultMessage="You can assign up to two categories to your integration."
+            />
+          }
+        >
+          <EuiComboBox
+            fullWidth
+            data-test-subj="editIntegrationFlyoutCategories"
+            aria-label="Select categories"
+            placeholder="Select categories"
+            selectedOptions={selectedCategories.map((category) => ({
```

Comment:
selectedOptions  could have a `value` and a `label` I think it could avoid the find you are doing when calling `updateCustomIntegration`